dp-download-service.nomad

```diff
@@ -117,7 +117,7 @@ job "dp-download-service" {
       }
 
       vault {
-        policies = ["dp-download-service", "read-psk"]
```

Comment:
Is the `read-psk` no longer required here?

Reply:
The content of "read-psk" has been moved to the dp-download-service-publishing policy